src/common/eeprom.h

```diff
@@ -52,11 +52,12 @@
     #define EEVAR_CONNECT_IP4   0x18 // X.X.X.X address encoded in uint32
     #define EEVAR_CONNECT_TOKEN 0x19 // 20char string
     #define EEVAR_LAN_HOSTNAME  0x1A // 20char string
+    #define EEVAR_CONNECT_PORT  0x1B // 0 - 65535 number of connect's port
 #endif                               // (EEPROM_FEATURES & EEPROM_FEATURE_LAN)
 
-#define EEVAR__PADDING 0x19 // 1..4 chars, to ensure (DATASIZE % 4 == 0)
+#define EEVAR__PADDING 0x1C // 1..4 chars, to ensure (DATASIZE % 4 == 0)
 
-#define EEVAR_CRC32 0x1a // uint32_t crc32 for
+#define EEVAR_CRC32 0x1D // uint32_t crc32 for
```

Comment:
how did you calculate crc? or how does it work?

Reply:
Same answer as the conversation above